project/AppDependencies.scala

```diff
@@ -2,8 +2,8 @@ import sbt.*
 
 object AppDependencies {
 
-  private lazy val hmrcBootstrapPlayVersion = "9.0.0"
-  private lazy val hmrcMongoPlayVersion     = "2.1.0"
+  private lazy val hmrcBootstrapPlayVersion = "9.2.0"
```

Comment:
Please update this to `9.3.0` and update the plugin which has been flagged to you by the PR builder. 

`9.3.0` will require `sbt-plugin` -> `3.0.5` rather than the suggestion included in the PR builder comment.

Please also bump `sbt-scoverage` -> `2.1.0`

Reply:
I've updated the dependancies

Reply:
I can't see the push yet